client/src/components/FootnoteList/FootnoteList.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { FootnoteList } from "./FootnoteList";
+
+export default {
+  title: "FootnoteList",
+  component: FootnoteList,
+};
+
+const Template = (args) => <FootnoteList {...args} />;
+
+// having issues with inputing year1 and year2 values
+const footnotes = [
+  {
+    text: "text1",
+    year1: "",
+    year2: "",
+    topic_keys: 1,
```

Comment:
`topic_keys` values should be an array of strings, should be text keys from `footnote_topics.yaml`.

Reply:
For now, I've initialized the topic_keys parameter to null so that it doesn't crash, but I'll note this.